src/views/StatsView.vue

```diff
@@ -115,19 +118,39 @@ export default {
       return this.seasons.map((season) => {
         return {
           title: season.name,
-          value: season
-        }
+          value: season,
+        };
       });
-    }
+    },
+  },
+  methods: {
+    checkAndSelectSeason(seasonId) {
```

Comment:
Is there ever the potential to call this without a seasonId?

Reply:
I don't believe so